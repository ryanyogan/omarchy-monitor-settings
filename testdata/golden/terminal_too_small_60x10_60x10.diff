Visual regression detected!

=== EXPECTED ===
                                                            
                                                            
                                                            
                                                            
                     [31mTerminal too small[0m                     
              [31mPlease resize to at least 80x20[0m               
                                                            
                                                            
                                                            
                                                            

=== ACTUAL ===
                                                            
                                                            
                                                            
                                                            
                     Terminal too small                     
              Please resize to at least 80x20               
                                                            
                                                            
                                                            
                                                            

=== DIFF INFO ===
Expected length: 627
Actual length: 609
Expected hash: 01867857216614f6624d9c69ff060351b0aaf2a37a750e0a5330f5636e9dc41c
Actual hash: 936b89060ee2f2a80c2ec0a3c48bd16df9e05580463f15d1c1521758d1ecb020
