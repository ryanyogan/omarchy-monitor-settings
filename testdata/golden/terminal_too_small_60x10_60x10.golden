# Visual Golden File
# Name: terminal_too_small_60x10
# Dimensions: 60x10
# Hash: 01867857216614f6624d9c69ff060351b0aaf2a37a750e0a5330f5636e9dc41c

                                                            
                                                            
                                                            
                                                            
                     [31mTerminal too small[0m                     
              [31mPlease resize to at least 80x20[0m               
                                                            
                                                            
                                                            
                                                            